packages/tempus/unit_test/Tempus_UnitTest_IMEX_RK.cpp

```diff
@@ -169,4 +173,205 @@ TEUCHOS_UNIT_TEST(IMEX_RK, AppAction)
+          {
+            const double X_i = get_ele(*(x), 0);
+            const double f_i = get_ele(*(xDot), 0);
+
+            if (stageNumber == 0) {
+              // X_i = 1, f_1 = 0
+              TEST_FLOATING_EQUALITY(X_i, -std::sqrt(3), relTol); // -sqrt(3)
+              TEST_FLOATING_EQUALITY(f_i,     1.0, relTol); // 1
+              TEST_FLOATING_EQUALITY(imp_time , 0.78867513459481275 , relTol); // 1
+              TEST_ASSERT(std::abs(time) < relTol);
+            } else if (stageNumber == 1) {
+              // X_i = , f_i =
+              TEST_FLOATING_EQUALITY(X_i,    3.0 - 3.0*std::sqrt(3.0), relTol); // -3sqrt(3) - 3
+              TEST_FLOATING_EQUALITY(f_i,    1.0, relTol); // 1
+              TEST_FLOATING_EQUALITY(time,    1.0, relTol);
+            } else {
+              TEUCHOS_TEST_FOR_EXCEPT( !(-1 < stageNumber && stageNumber < 2));
+            }
+
+            break;
+          }
+        case StepperRKAppAction<double>::END_STEP:
+          {
+            const double x_1 = get_ele(*(x), 0);
+            time = workingState->getTime();
+            TEST_FLOATING_EQUALITY(x_1,  -(6.0*std::sqrt(3) - 11.0/2.0), relTol);   // -( 6sqrt(3) - 11/2)
+            TEST_FLOATING_EQUALITY(time,     1.0, relTol);
+            TEST_FLOATING_EQUALITY(dt,       1.0, relTol);
+            TEST_COMPARE(stageNumber, ==, -1);
+
+            if (stepper->getUseEmbedded() == true) {
+              TEST_FLOATING_EQUALITY(workingState->getTolRel(), 1.0, relTol);
+              TEST_ASSERT(std::abs(workingState->getTolAbs()) < relTol);
+              // e = 0 from doxygen above.
+              TEST_ASSERT(std::abs(workingState->getErrorRel()) < relTol);
+            }
+
+          }
+
+      }
+
+
+    }
+
+  private:
+
+    Teuchos::FancyOStream & out;
+    bool & success;
+};
+
+// ************************************************************
+// ************************************************************
+TEUCHOS_UNIT_TEST(IMEX_RK, IMEX_RK_Modifier)
+{
+
+  Teuchos::RCP<const Thyra::ModelEvaluator<double> > explicitModel = rcp(new Tempus_Test::DahlquistTestModel<double>(1.0, true));
+  Teuchos::RCP<const Thyra::ModelEvaluator<double> > implicitModel = rcp(new Tempus_Test::DahlquistTestModel<double>(2.0, true));
+
+  auto model = rcp(new Tempus::WrapperModelEvaluatorPairIMEX_Basic<double>(
+                                             explicitModel, implicitModel));
+
+  auto modifier = rcp(new StepperRKModifierIMEX_TrapezoidaTest(out, success));
+
+
+  // Default construction.
+  auto stepper = rcp(new Tempus::StepperIMEX_RK<double>());
+  stepper->setModel(model);
+
+  // Default values for construction.
+  auto solver = rcp(new Thyra::NOXNonlinearSolver());
+  solver->setParameterList(Tempus::defaultSolverParameters());
+
+  bool useFSAL              = stepper->getUseFSAL();
+  std::string ICConsistency = stepper->getICConsistency();
+  bool ICConsistencyCheck   = stepper->getICConsistencyCheck();
```

Comment:
I still get an error on this ...
`/Users/ccober/myOffice/Trilinos_Second/packages/tempus/unit_test/Tempus_UnitTest_IMEX_RK.cpp:336:8: error: unused variable 'ICConsistencyCheck' [-Werror,-Wunused-variable]
  bool ICConsistencyCheck   = stepper->getICConsistencyCheck();
       ^
1 error generated.
`

Reply:
My apologies. I turned the warnings for my config script. I think I've caught them now